roles/fluentd_master/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+embedded_etcd: "{{ openshift.master.embedded_etcd }}"
```

Comment:
I'm not sure the related checks are even needed anymore with the test in the openshift_master role to verify that the api server is up before continuing.

As an aside, we should update for the proper package if it is available in the shipped repos as opposed to the public repo that it is currently being pulled from.